introspection_request_handler.go

```diff
@@ -167,13 +167,15 @@ func (f *Fosite) NewIntrospectionRequest(ctx context.Context, r *http.Request, s
 		Active:          true,
 		AccessRequester: ar,
 		TokenType:       tt,
+		AccessTokenType: att,
 	}, nil
 }
 
 type IntrospectionResponse struct {
 	Active          bool            `json:"active"`
 	AccessRequester AccessRequester `json:"extra"`
-	TokenType       TokenType       `json:"token_type,omitempty"`
+	TokenType       TokenType       `json:"token_use,omitempty"`
```

Comment:
I think we could change this for readability purposes in a breaking fashion so that we have:

```suggestion
	TokenUse       TokenUse       `json:"token_use,omitempty"`
	AccessTokenType Type `json:"token_type,omitempty"`
```